src/authentication/authentication-hapi-plugin.ts

```diff
@@ -248,7 +248,13 @@ class AuthenticationHapiPlugin extends HapiPlugin {
       const teamTokenResult = await teamTokenQuery(this.db, { teamId: team.id });
       const teamToken =
         (teamTokenResult && teamTokenResult.length) ? teamTokenResult[0].token : undefined;
-      return reply({ ...team, 'invitation-token': teamToken });
+      return reply({
+        id: team.id,
+        name: team.id,
```

Comment:
Shouldn't this be `team.name`? This could be tested, along with all the properties included here, in the unit test.